rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import copy from 'rollup-plugin-copy';
+import del from 'rollup-plugin-delete';
+import dynamicImportVars from '@rollup/plugin-dynamic-import-vars';
+import glob from 'glob-all';
+import resolve from '@rollup/plugin-node-resolve';
+
+const jsGlob = [
+	'@(components|controllers|directives|helpers|mixins|templates)/**/*.js',
+	'!**/*@(test|axe|visual-diff).js',
+];
```

Comment:
This selects all JS files in our "main" directories, and filters out any `test.js`, `axe.js`, and `visual-diff.js`.

I confirmed that there are no cases where any of the above files are being imported in HTML.